components/core/tests/test-ffi_SchemaTree.cpp

```
@@ -20,25 +18,26 @@ namespace {
+ * the ones specified in the locator.
  */
 [[nodiscard]] auto check_node(
         SchemaTree const& schema_tree,
         SchemaTree::NodeLocator locator,
-        SchemaTreeNode::id_t expected_id
+        SchemaTree::Node::id_t expected_id
 ) -> bool;
 
 auto insert_node(
         SchemaTree& schema_tree,
         SchemaTree::NodeLocator locator,
```

cppcheck

[performance] Function parameter 'locator' should be passed by const reference.

(passedByValue)